### PR TITLE
US-37: associer une epreuve a un test

### PR DESCRIPTION
**Pull request by mackwic**

### CODE REVIEW

**Comment by mackwic** [comment]
So, tests are broken and I don't know why (yet).

I made the migration to factories and, in the debugger, it works great. I surely made a logic error somewhere in the wiring. Happy to get some stranger eyes on the code to help me if you have time.

**Comment by mackwic** [comment]
Good for review / merge. I moved the refactoring commits in branch [`refacto-37-associer-une-epreuve-a-un-test`](https://github.com/sgmap/pix-live/commits/refacto-37-associer-une-epreuve-a-un-test)

File: tests/unit/routes/courses/challenge-preview-test.js [hunk 1]
```diff
@@ -16,5 +16,7 @@ describeModule(
       let route = this.subject();
       expect(route).to.be.ok;
     });
+
+    it('')
```
Comment (jbuget) [inline tests/unit/routes/courses/challenge-preview-test.js]: to be deleted

File: app/serializers/course.js [hunk 2]
```diff
@@ -11,6 +12,10 @@ export default AirtableSerializer.extend({
         fields.duration = fields['Durée'];
         fields.challenges = fields['Épreuves'];
 
+        if (_.isArray(fields.challenges)) {
+          fields.challenges.reverse();
```
Comment (jbuget) [inline app/serializers/course.js:16]: bien vu ^^

**Pull request merged**